source/administration/concepts.rst

```diff
@@ -0,0 +1,162 @@
+Specifically, clients must present a valid access key and secret key to access any S3 or MinIO administrative API, such as ``PUT``, ``GET``, and ``DELETE`` operations. 
+
+MinIO then checks that authenticated users or clients have *authorization* to perform actions or use resources on the deployment. 
+MinIO uses :ref:`Policy-Based Access Control (PBAC) <minio-policy>`, where each policy describes one or more rules that outline the permissions of a user or group of users. 
+MinIO supports S3-specific :ref:`actions <minio-policy-actions>` and :ref:`conditions <minio-policy-conditions>` when creating policies. 
+
+By default, MinIO *denies* access to actions or resources not explicitly referenced in a user's assigned or inherited policies.
+
+MinIO provides an access management feature as part of the software.
+Alternatively, you can configure MinIO to authenticate with one of several external IAM providers using either :ref:`Active Directory/LDAP <minio-external-identity-management-ad-ldap>` or :ref:`OpenID/OIDC <minio-external-identity-management-openid>`.
+
+
+How does MinIO secure data?
+---------------------------
+
+MinIO supports methods that encode objects while on disk (encryption-at-rest) and during transition from one location to another (encryption-in-transit, or "in flight").
+When enabled, MinIO utilizes :ref:`server-side encryption <minio-encryption-overview>` to write objects in an encrypted state.
+To retrieve and read an encrypted object, the user must have appropriate access privileges and also provide the object's decryption key.
+
+MinIO supports **Transport Layer Security** (TLS) versions 1.2 and 1.3 encrypting objects.
+TLS replaces the previously used Secure Socket Layer (SSL) method that has since been deprecated.
+The TLS standard, maintained by the Internet Engineering Task Force (IETF), provides the standards used by internet communications to support encryption, authentication, and data integrity.
+
+The process of authenticating a user and verifying access to objects is known as the `TLS Handshake`.
+Once authenticated, TLS provides the cipher to encypt and then decrypt the transfer of information from the server to the requesting client.
+
+MinIO supports several methods of :ref:`Server-Side Encryption <minio-encryption-overview>`.
+
+Can I organize objects in a folder structure within buckets?
+------------------------------------------------------------
+
+MinIO is a flat object store, meaning there is no structure to a deployment other than the bucket(s).
+However, MinIO does allow for using a prefixing scheme with a ``/`` delimitor that can mimic a folder structure from traditional file systems.
+
+Prefixing involves prepending the name of an object with a fixed string.
+When you use the same prefix string on multiple objects, MinIO identifies those as similar or grouped objects.
```

Comment:
I think calling MinIO a 'flat object store', while technically true, may be misleading. I _think_ the distinction is more that:

- MinIO supports any arbitrary hierarchy of objects in a bucket
- You do not create "subfolders" - you use prefixes as part of the object name
- MinIO infers from that full path what the "subfolders" are
- Multiple objects with similar prefix paths "look" like folder/subfolder relationships from the POSIX world
- All of this is automatic.

Reply:
I'm not able to be more prescriptive *yet*. The prefixing stuff is vital, and hard to explain.

Reply:
The water is definitely muddied because we use prefixes, but call them folders and subfolders (very much against Amazon there). I reworked this section.